shared/locales/langpack/desc.en-US.properties

```diff
@@ -0,0 +1,24 @@
+# This file provides localization string for the Language Pack manifest
+
+###
+# The name of the langpack and its description will be displayed to the user
+# when browsing the list of language packs on the marketplace and during
+# installation.
+#
+# IMPORTANT: these strings need to be adapted to your language, not simply 
+# translated. "Locale" needs to be replaced with the localized version 
+# of your language name: "Italiano" for Italian, "Polski" for Polish, etc. 
+# Add the region if it's relevant for your locale, e.g. "English (US)".
+#
+# brandShortName will be taken from shared/locales/branding/
+###
+langpack.name = English (US) Language Pack
+langpack.description = A package providing an American English localization of {{ brandShortName }}.
+langpack_enUS.name = English (US) Language Pack
+langpack_enUS.description = A package providing an American English localization of {{ brandShortName }}.
```

Comment:
Nit, my feeling is that it's "localization for.." instead of "localization of..".